### PR TITLE
feat: Add python solution for euler#53

### PR DESCRIPTION
**Pull request by ahdeshpande**

### CODE REVIEW

File: problems/euler/52/52.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+"""
+Euler Problem: 52
```
Comment (mre) [inline problems/euler/52/52.py]: wait, the pull request title says "Add python solution for euler#53", but this is the solution for 52. Was that a mistake?

**Comment by mre** [comment]
wait, the pull request title says "Add python solution for euler#53", but this is the solution for 52. Was that a mistake?

**Comment by ahdeshpande** [comment]
I'll submit a new PR if that's okay. Let me know @mre.

**Comment by mre** [comment]
Yeah, if you have the time and you'd like to do it sure, go ahead. 👍

**Pull request closed without merging**